infra/core/ai/bingSearch/bingSearch.tf

```diff
@@ -23,4 +23,10 @@ resource "azurerm_resource_group_template_deployment" "bing_search" {
   # this ensures the keys are up-to-date
   name            = "bingsearch-${filemd5(local.arm_file_path)}"
   deployment_mode = "Incremental"
+}
+
+resource "azurerm_key_vault_secret" "search_service_key" {
```

Comment:
update the name here to match the context.

Reply:
Changed